lib/scholars_archive/triple_powered_properties/has_url_validator.rb

```diff
@@ -11,6 +11,7 @@ def validate(record)
       return if record.triple_powered_properties.empty?
 
       record.triple_powered_properties.each do |prop|
+        return if prop[:skip_validation] && record[prop].empty? 
```

Comment:
So we have to check if also the field is empty because if there is a value added to the field then we definitely want to validate. So we will skip iff the skip_validation is set to true and the field is empty.